p-token/tests/approve.rs

```diff
@@ -85,7 +85,7 @@ async fn approve() {
 }
 
 // #[tokio::test]
-async fn approve_invalid_src() {
+async fn _approve_invalid_src() {
```

Comment:
Are these functions called internally? If not, maybe it makes more sense to mark them `pub` and leave them without the underscore.